core/server/master/src/main/java/alluxio/master/file/meta/InodeTreePersistentState.java

```diff
@@ -284,8 +301,10 @@ public void applyAndJournal(Supplier<JournalContext> context, MutableInode<?> in
       applyCreateInode(inode);
       context.get().append(inode.toJournalEntry());
     } catch (Throwable t) {
-      ProcessUtils.fatalError(LOG, t, "Failed to apply %s", inode);
-      throw t; // fatalError will usually system.exit
+      ProcessUtils.fatalErrorWithCheck(mTolerantCorruption, LOG, t, "Failed to apply %s", inode);
```

Comment:
can we move the error handling to a common method?

Reply:
we cannot move the `throw t` to a method, thanks

Reply:
I think we can do something like
```java
handleJournalReplayFailure(masterName, journalEntry, t);
throw t;
```

Reply:
Done!
change to something like
```
handleJournalReplayFailure(masterName, journalEntry, t);
if (!tolerrant) {
  throw t;
}
```